app/Picture.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from PIL import Image
+
+if TYPE_CHECKING:
+    from Tile import Tile
+
+
+class Picture:
+    def __init__(self) -> None:
+        self.image: Image.Image | None = None
+        self.is_solved = False
+        self.tiles: dict[int:Tile] = {}  # init so that autocomplete works
+        self.tile_order = []
+        self.scramble_type = None
+
+    def load(self, img_path: str) -> None:
+        """Loads an image from a file path.
+
+        :param img_path: Absolute path to the image file
+        """
+        self.image = Image.open(img_path)
+
+    def save(self, img_path: str, scramble_type: str) -> None:
```

Comment:
This method will need to be changed for each type of scrambling we come up with. Fine for now, since it's only two types.

This is a candidate for refactoring if we decide to add more types of scrambling. The [strategy pattern](https://refactoring.guru/design-patterns/strategy) is a good fit here.

Reply:
Created an issue for this to be refactored later #10